W-14927671  Pdk ga branch gr (#464)
mulesoft/docs-gateway: pdk/1.0/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc

= Debugging Deployed Custom Policies
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images

> If a custom policy deployed on a Flex Gateway is malfunctioning, there are some different methods to debug the policy:

Reviewer: ```suggestion
You can debug a malfunctioning policy via the following methods:
```

* <<debug-environment>>
* <<integration-testing>>
* <<debug-logs>>

[[debug-playground]]

> == Use PDK Debugging Playground

Reviewer: ```suggestion
== Use the PDK Debugging Playground
```
How does marketing want the debugging playground positioned? (With a "the" or without?)
